lang/cs/Org.Apache.REEF.Wake/Remote/Impl/DefaultRemoteManager.cs

```diff
@@ -144,20 +143,56 @@ public IObserver<T> GetRemoteObserver(IPEndPoint remoteEndpoint)
             ProxyObserver remoteObserver;
             if (!_cachedClients.TryGetValue(remoteEndpoint, out remoteObserver))
             {
-                TransportClient<IRemoteEvent<T>> client = 
-                    new TransportClient<IRemoteEvent<T>>(remoteEndpoint, _codec, _observerContainer);
-                var msg = string.Format("NewClientConnection: Local {0} connected to Remote {1}",
-                    client.Link.LocalEndpoint.ToString(), 
-                    client.Link.RemoteEndpoint.ToString());
-                LOGGER.Log(Level.Info, msg);
-
-                remoteObserver = new ProxyObserver(client);
+                remoteObserver = CreateRemoteObserver(remoteEndpoint);
                 _cachedClients[remoteEndpoint] = remoteObserver;
             }
 
             return remoteObserver;
         }
 
+        public IRemoteObserver<T> GetUnmanagedRemoteObserver(RemoteEventEndPoint<T> remoteEndpoint)
```

Comment:
I am not. Just added same methods as we had them before.

-----Original Message-----
From: "Andrew Chung" <notifications@github.com>
Sent: ‎3/‎29/‎2016 4:14 PM
To: "apache/reef" <reef@noreply.github.com>
Cc: "Boris Shulman" <shulmanb@gmail.com>
Subject: Re: [apache/reef] [REEF-1142] Making caching configurable in theremote managers (#905)

In lang/cs/Org.Apache.REEF.Wake/Remote/Impl/DefaultRemoteManager.cs:
>                  _cachedClients[remoteEndpoint] = remoteObserver;
>              }
>  
>              return remoteObserver;
>          }
>  
> +        public IRemoteObserver<T> GetUnmanagedRemoteObserver(RemoteEventEndPoint<T> remoteEndpoint)
Why this type of parameter? What does this provide over IPEndpoint?
—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub